PyStemmusScope/global_data_selection.py

```diff
@@ -0,0 +1,231 @@
+from pathlib import Path
+from typing import Dict
+from typing import List
+from typing import Tuple
+from typing import Union
+import numpy as np
+import xarray as xr
+from . import utils
+from . import variable_conversion as vc
+
+
+TIMESTEP = "1800S"
```

Comment:
`TIMESTEP = "1800S" ` is also defined in `forcing_io`. Can we remove it from here and consider it as an input argument for functions in `global_data_selection`?

Reply:
We could make it an argument I guess, yes. I'll make the modifications.

Reply:
Done! 👍